doc/Type/Compiler.pod6

```diff
@@ -31,7 +34,8 @@ It's empty, but it might contain the codename for specific releases.
 
     method build-date()
 
-Up to version 2019.03.1, it returns the date when it was built.
+Up to Rakudo version 2019.03.1, hence only with C<$*PERL.compiler>,
```

Comment:
Don't really understand this sentence. The hence, mainly.

Reply:
yes it was a bit cryptic. I simply removed this :-)